cli/cli.py

```diff
@@ -393,6 +442,35 @@ def check_error_measure_option_presence(task: str | None, error_measure: str | N
         sys.exit(1)
 
 
+def parse_tables_list_file(file: click.File) \
+        -> list[tuple[str, str, bool]]:
+    try:
+        result = []
+        for line_num, line in enumerate(file.readlines()):
```

Comment:
```suggestion
        for line_num, line in enumerate(file.readlines(), start=1):
```
Text editors normally count lines from 1